lib/katello/repository_types/docker.rb

```diff
@@ -1,5 +1,7 @@
 Katello::RepositoryTypeManager.register(::Katello::Repository::DOCKER_TYPE) do
   service_class Katello::Pulp::Repository::Docker
+  pulp3_service_class Katello::Pulp3::Repository::Docker
+  pulp3_plugin 'pulp_docker'
 
```

Comment:
Does docker content not have a publication?

Reply:
Nvm..missed your earlier comment..Also, you could remove pry as it's not letting your tests run..

Reply:
It doesn't, no

Reply:
> Nvm..missed your earlier comment..Also, you could remove pry as it's not letting your tests run..

Oh my bad, I missed that!